msk/actions/create.py

```diff
@@ -269,6 +331,8 @@ def initialize_template(self, files: set = None):
             )),
             ('.git', lambda: git.init())
         ]
+        if self.manifest:
+            skill_template.append(("manifest.yml", lambda: self.manifest))
```

Comment:
Would it be simpler to shift the call of `dependencies` to the `skill_template` itself? Because the function returns `None` if the user selects `No` then no file will be created in that instance anyway.
```
skill_template = [
            ('', lambda: makedirs(self.path)),
            ('vocab', self.add_vocab),
            ('dialog', self.add_dialog),
            ('__init__.py', lambda: self.init_file),
            ('README.md', lambda: self.readme),
            ('LICENSE.md', self.license),
            ('.gitignore', lambda: gitignore_template),
            ('settingsmeta.yaml', lambda: settingsmeta_template.format(
                capital_desc=self.name.replace('-', ' ').capitalize()
            )),
            ('manifest.yml', lambda: self.dependencies),
            ('.git', lambda: git.init())
        ]
```